src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -455,11 +456,14 @@ struct LottiePathSet : LottieProperty
         return (*frames)[frames->count];
     }
 
-    bool operator()(float frameNo, Array<PathCommand>& cmds, Array<Point>& pts, Matrix* transform)
+    bool operator()(float frameNo, Array<PathCommand>& cmds, Array<Point>& pts, Matrix* transform, float roundness)
     {
         if (!frames) {
-            _copy(value, cmds);
-            _copy(value, pts, transform);
+            if (roundness > ROUNDNESS_EPSILON) _handleCorners(cmds, pts, transform, roundness);
```

Comment:
roundness can also be applied to the keyframe data. Consequently, the logic should be adjusted for additional cases as well. Refer to lines 470, 476, 484, and the final one at 510 as the interpolated result.

Reply:
ofc